workers/unity/Packages/com.improbable.gdk.gameobjectcreation/Tests/WorkerEntityLinkingTests.cs

```diff
@@ -0,0 +1,82 @@
+using System.Runtime.CompilerServices;
```

Comment:
What are we using this for?

Reply:
Forgetting to clean up weird things added by the IDE while autocompleting as usual